modules/test/playwright/tests/object-web/objectValidations.spec.ts

```diff
@@ -20,19 +20,29 @@ export const test = mergeTests(
 	objectPagesTest
 );
 
-let objectDefinition: ObjectDefinition;
+let objectDefinition1: ObjectDefinition;
+let objectDefinition2: ObjectDefinition;
 
 test.beforeEach(async ({apiHelpers}) => {
-	const newObjectDefinition =
+	const newObjectDefinition1 =
 		await apiHelpers.objectAdmin.postRandomObjectDefinition({
+			objectFolderExternalReferenceCode: 'default',
 			status: {code: 0},
 		});
 
-	objectDefinition = newObjectDefinition;
+	const newObjectDefinition2 =
+		await apiHelpers.objectAdmin.postRandomObjectDefinition({
+			objectFolderExternalReferenceCode: 'default',
+			status: {code: 0},
+		});
+
+	objectDefinition1 = newObjectDefinition1;
+	objectDefinition2 = newObjectDefinition2;
```

Comment:
I think that even if we don't use it, it is safer to create here than in the test, we are trying to avoid creating a cleanup for each test because if the test fails for some reason in the middle of the test, the cleanup created within the test will not be executed, generating entities useless on the portal.